application/src/main/java/org/togetherjava/tjbot/commands/tophelper/TopHelpersMessageListener.java

```diff
@@ -85,4 +70,26 @@ private void addMessageRecord(MessageReceivedEvent event) {
             .setMessageLength(messageLength)
             .insert());
     }
+
+    boolean shouldIgnoreMessage(MessageReceivedEvent event) {
+        MessageChannelUnion channel = event.getChannel();
+
+        return event.getAuthor().isBot() || event.isWebhookMessage() || !isHelpThread(channel);
```

Comment:
```suggestion
        return event.getAuthor().isBot() || event.isWebhookMessage() || !isHelpThread(event.getChannel());
```
I personally see no reason for the variable, don't know if you added this due to prior CR or just added it yourself?

Reply:
huh might be a leftover of debugging